doc/neps/roadmap.rst

```diff
@@ -122,6 +125,13 @@ We intend to write a NEP covering the support levels we provide and what is
 required for a platform to move to a higher tier of support, similar to
 `PEP 11 <https://peps.python.org/pep-0011/>`__.
 
+Further consistency fixes to promotion and scalar logic
+```````````````````````````````````````````````````````
+NumPy 2 fixed many issues around promotion especially with respect to scalars.
+We plan to continue slowly fixing remaining inconsistencies.
+For example, NumPy converts 0-D objects to scalars and some promotions
```

Comment:
```suggestion
For example, NumPy converts 0-D objects to scalars, and some promotions
```

Reply:
Do we (you, or anyone else) actually plan to do anything about 0-D arrays vs. scalars? If not, we shouldn't mention it here.

Reply:
Not promising to do this right now, but that is true for most points on the list.

So yes, I think this is much more concrete than most points.  In fact, it is even mostly done, since I did most (maybe all) of the foundational work in NumPy 2!  (and one of that was another small compat break ;).)